packages/playground/package.json

```diff
@@ -0,0 +1,78 @@
+{
+    "name": "@prezly/playground",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@popperjs/core": "^2.6.0",
+        "@prezly/docx-cleaner": "^0.1.8",
+        "@prezly/events": "^1.0.2",
+        "@prezly/linear-partition": "^1.0.0",
+        "@prezly/progress-promise": "^1.0.0",
+        "@prezly/slate-commons": "^0.1.8",
+        "@prezly/slate-editor": "^0.1.8",
+        "@prezly/slate-hyperscript": "^0.1.8",
+        "@prezly/slate-lists": "^0.1.8",
+        "@prezly/slate-types": "^0.1.8",
+        "@prezly/uploadcare": "^1.0.0",
+        "json-stable-stringify": "^1.0.1",
+        "lodash": "^4.17.21",
+        "moment": "^2.29.1",
+        "rangefix": "^0.2.9",
+        "react": "^16.14.0",
+        "react-dom": "^16.14.0",
+        "react-bootstrap": "=0.32.4",
+        "react-draggable": "^3.3.2",
+        "react-overlays": "^0.8.3",
+        "react-popper": "^2.2.4",
+        "react-portal": "^4.2.1",
+        "react-router-dom": "=5.1.2",
+        "react-scripts": "^4.0.3",
+        "react-use": "^17.2.1",
+        "shave": "^2.5.10",
+        "slate": "0.59.0",
+        "slate-history": "0.59.0",
+        "slate-react": "0.59.0",
+        "uploadcare-widget": "=3.7.7",
+        "uuid": "^8.3.2",
+        "validator": "^13.5.2"
+    },
+    "devDependencies": {
+        "@types/react-dom": "^17.0.9",
+        "typescript": "^4.3.5",
+        "customize-cra": "^1.0.0",
+        "react-app-rewired": "^2.1.8",
+        "@types/classnames": "^2.2.11",
+        "@types/json-stable-stringify": "^1.0.32",
+        "@types/lodash": "^4.14.167",
+        "@types/react": "^16.14.2",
+        "@types/react-bootstrap": "^0.32.19",
+        "@types/react-overlays": "^1.1.3",
+        "@types/react-portal": "^4.0.2",
+        "@types/react-router-dom": "=5.1.2",
+        "@types/uuid": "^3.4.5",
+        "@types/validator": "^13.1.3",
+        "bootstrap-sass": "^3.4.1"
+    },
+    "scripts": {
+        "start": "react-app-rewired start",
+        "clean": "rimraf build/ node_modules/"
+    },
```

Comment:
In all other `package.json` files in this repository, `scripts` are above `*dependencies`

Reply:
9780600